swagger.yaml

```diff
@@ -4463,7 +4459,11 @@ definitions:
       schedulable_masters:
```

Comment:
This was removed from create, but left in update?
Seems strange.

Reply:
The user will be able to change the scheduling on master's side (update the cluster object) when the number of hosts is 5 or more.